docs-ref-services/latest/resourcemanager-search-readme.md

```diff
@@ -1,15 +1,18 @@
 ---
-title: Azure Resource Manager Cognitive Search client library for Java
+title: Azure Resource Manager Azure AI Search client library for Java
```

Comment:
@HeidiSteen - There is a merge conflict in this file that prevents merging. Otherwise this PR looks good.